README.rst

```diff
@@ -210,10 +214,29 @@ Staff Grading
    student will be able to view the file along with her grade.
 
 .. figure:: https://raw.githubusercontent.com/mitodl/edx-sga/screenshots/img/screenshot-graded.png
-   :alt: Staff view of grading grid after a submission has been graded
+   :alt: Instructor view of grading grid after a submission has been graded.
 
-7. Course staff can enter grades, but only Instructors can approve grades.
-   Grades must be approved for the student to view them.
+7. Course staff can enter grades, but only Instructors can approve grades (and
+   remove grades). 
+
+   When a course staff member enters a grade, the grade will be followed by the
+   text: :code:`(Awaiting instructor approval)`. The student will not see this 
```

Comment:
It took me awhile to find the ```Awaiting instructor approval``` text.  Consider mentioning that this text appears in the Grade column.